### DISCUSSION TITLE
Annotate PDFs- make additional layers different sizes

### DISCUSSION THREAD

**Discussion by AshleyDiss** [xournalpp/xournalpp#3807] [Q&A]
Hello,

I am trying to annotate PDF articles from journals, which are way too dense in content to write on the PDF. I have been trying to find a way to write on the sides of the pages through an additional layer that is wider than the pdf layer, but I can't find a way to change the size. Is there a way to do this or some other approach people use to annotate content dense pdf's in xournal++?

**Comment by netlimpopo** [comment]
Hello!
One way is to insert a whole new page after each pdf page. If one views the document in the 2 page layout the pdf page and the blank page apear next to each other. You can also change the page Format if you want a leaner paper size, so it fits better next to each other. 
Layers only realy add a layer on top, which won't really work. 
Currently there is work going on to enable an infinite canvas which might also solve solve the issue.

**Reply by maylor-afk** [reply]
Hello!
Is it already possible to use the infinite canvas?

**Reply by netlimpopo** [reply]
Hello! Unfortunately no. There is no infinite canvas implemented yet. Some refactoring of the code has been happening, which is necessary for this step. But it is challenging task and will take some time. 
Sorry.
